src/Chapter1.hs

```diff
@@ -635,8 +645,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n < 10 = n
```

Comment:
This case covers _all_ numbers below ten, but what about -10? It will go into this case always, so we need to do something about negatives as well here 🙂 

Btw, we allowed failure on CI for advanced tasks, that is why you are not notified about that...

Reply:
I lost this case, I will try to solve it. Thank you!